### DISCUSSION TITLE
CbzMage no longer works with the latest Kdl (v2.2.0)

### DISCUSSION THREAD

**Discussion by ToofDerling** [yegon666/CbzMage#35] [Announcements]
Kdl 2.2.0 introduces drm for downloaded comics, when you run CbzMage on the download it will fail with a MOBI identifier error. There's no way to remove Kdl drm - the "big secret" of CbzMage was that, until now, there was no drm at all for Cmxlgy books.

Kdl 2.0.0 still works though (and you can find downloads of that version with a little searching). I downgraded to 2.0.0 and bought a couple of books, they both downloaded and converted fine. Note that if you downgrade you should turn off the sneaky "Automatically install updates" option in Kdl.

Of course 2.0.0 will only work until you're forced to upgrade Kdl to download the comics you bought, Hopefully that will be a while...

**Comment by ToofDerling** [comment]
I have tested Kdl 2.1.0, it also works. 1.3.9 is confirmed working [here](https://github.com/ToofDerling/CbzMage/issues/34), but 1.3.8 requires you to upgrade to download newly released comics. 

So the working versions are likely 1.3.9 - 2.1.0 which can all be downloaded [here](https://kindle-for-pc.en.uptodown.com/windows/versions)

**Comment by zimdog1701** [comment]
Thanks - Can you provide corresponding info for those of us on the Mac side? My own version of Kindle Classic for Mac is 1.40.3, last updated 2/20/2022, but the app is now reporting that "support" was ending by October 31, 2023.  Thus far it's not forcing an update.

I did find that previous versions of Kdl for Mac can be located [here.](https://kindle-mac.en.uptodown.com/mac)

**Comment by rafial** [comment]
For the record, experimented 1.40 and 1.39 version of the Kdl for Mac app found at the link in the previous post, and both seemed to be downloading .kcr, so don't seem to be useful with CbzMage.
